static/styles/settings.css

```diff
@@ -38,6 +38,35 @@ h3,
     }
 }
 
+/* dropdown design */
+.dropdown-wrap {
+    position: relative;
+    width: fit-content;
+
+    select {
+        appearance: none;
+        margin: 0;
+    }
+
+    &::before,
+    &::after {
+        content: "";
+        position: absolute;
+        top: 50%;
+        right: 15px;
+        width: 2px;
+        height: 9px;
+        background: hsl(0, 0%, 58%);
+        border-radius: 2px;
+        transform: translate(-50%, -50%) rotate(40deg);
+    }
+
+    &::after {
+        right: calc(15px + 7px);
+        transform: translate(50%, -50%) rotate(-40deg);
```

Comment:
@Taiki-Choda can you add comments on why do you think these numbers would work in all scenarios or how you calculated them?